herring/puzzles/models.py

```diff
@@ -17,9 +38,11 @@ def __str__(self):
 
     class Meta:
         ordering = ['number']
+    class Json:
+        include_fields = ['name', 'number']
```

Comment:
It could.  One potential issue is that you have to be careful about not referencing relations if they can wind up being circular, as `puzzle_set` and `parent` would wind up being.  Django models have a *bunch* of attributes, and you really only want the Fields.

Another is that this approach lets you get things serialized top down (which is more what you want for React) when the models foreign-key bottom up, by allowing you to specify `puzzle_set` but not `parent`.  Patchset with that approach incoming; I'll let you decide if you still want to serialize all fields by default after that.